view/card/mixin.pug

```diff
@@ -59,3 +59,11 @@ mixin card(type)
         })
       .card_range
         +range('a', ALPHA_MAX, ALPHA_STEP, 100)
+
+    else if type === 'model'
```

Comment:
Let’s remove the card and just add the model above the cards. We will find another spot.

Reply:
I moved the card to the model's mixin. I think some there are few more small changes that can be made to make css and page structure more consistent. Working on it